compiler/luci/pass/src/QuantizedModelVerifier.cpp

```diff
@@ -22,14 +22,75 @@
 #include <luci/IR/CircleNodes.h>
 #include <luci/IR/CircleNodeVisitor.h>
 
+using LayerInfoMap = std::unordered_map<std::string, luci::LayerInfo *>;
+
 namespace luci
 {
 
+LayerInfoMap QuantizedModelVerifier::create_layer_info_map(loco::Graph *g) const
```

Comment:
This function can be improved just like https://github.com/Samsung/ONE/pull/8555#discussion_r818311347 :)

Reply:
As mentioned https://github.com/Samsung/ONE/pull/8555#discussion_r818339492, the code raises an error.

Reply:
Ah, I missed the comment sorry :(
I thought that all nodes in the graph should be appeared in `info_by_name`, but all names in `info_by_name` should be appeared in the graph T.T
I will post another PR to make it better later :)
Thanks for pointing out!